### PR TITLE
Data downloads

### PR DESCRIPTION
**Pull request by JuliaWinn**
Added links for the following data feeds:

-people visiting each site, realtime
-visits to each site over the last 30 days
-visits broken down by country over the last 90 days
-visits broken down by city over the last 90 days
-top 100 downloads in realtime
-top 100 downloads over the past 30 days

Let me know if I missed any.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
-	sass --watch $(scss):$(css)
-
 production:
 	bundle exec jekyll build
 
 dev:
 	bundle exec jekyll serve --watch --config=_config.yml,_development.yml
-
-clean:
-	rm -f $(css)
```
Comment (konklone) [inline Makefile]: Why are all these lines deleted...?

File: _config.yml [hunk 2]
```diff
@@ -29,3 +29,7 @@ repo_name: analytics.usa.gov
 
 # twitter information
 twitter: usgsa
+
+sass:
+  sass_dir: sass
+  style: nested
```
Comment (konklone) [inline _config.yml]: Are we also changing the Sass directory structure as part of this?

File: _development.yml [hunk 3]
```diff
@@ -4,4 +4,4 @@ url: http://localhost:4000
 #   analytics --output=[dir]
 # and running from [dir]:
 #   serve
-data_url: http://localhost:3000
+data_url: http://localhost:3000/
```
Comment (konklone) [inline _development.yml]: Why the trailing slash? I think this is unnecessary.

**Comment by konklone** [comment]
There are too many unrelated commits in this pull request. Can you resubmit a PR with only the data download changes?

**Comment by JuliaWinn** [comment]
Happily, I'm not sure how the other changes got in there.

**Pull request closed without merging**